InscryptionCommunityPatch/Card/Part3StatIcons.cs

```diff
@@ -0,0 +1,61 @@
+using DiskCardGame;
+using HarmonyLib;
+using UnityEngine;
+
+namespace InscryptionCommunityPatch.Card;
+
+[HarmonyPatch]
+internal static class Part3StatIcons
+{
+    [HarmonyPatch(typeof(DiskCardGame.Card), "RenderCard")]
+    [HarmonyPrefix]
+    private static void UpdateLiveRenderedCard(ref DiskCardGame.Card __instance)
+    {
+        CardStatIcons icons = __instance.gameObject.GetComponentInChildren<CardStatIcons>();
+        icons.attackIconRenderer.gameObject.transform.localPosition = new(0.39f, 0.033f, 0f);
+        icons.attackIconRenderer.gameObject.transform.localScale = new (0.2f, 0.2f, 1f);
+        icons.attackIconRenderer.material.color = GameColors.Instance.brightBlue;
```

Comment:
You can extract the setting logic to it's own method since almost everything is the same, with the only difference being the layer being set. Could probably make that an optional param.

Reply:
There's actually something subtlely different in each of the three of these; some of them have to have the x-position of the icons changed, some have the renderers disabled, and some are being put into different layers.